src/main/java/use_case/delete_clothing_item/DeleteOutputData.java

```diff
@@ -0,0 +1,4 @@
+package use_case.delete_clothing_item;
+
+public record DeleteOutputData() {
+}
```

Comment:
Do we actually need it if it's empty? What output data would there be if we're just deleting a record from a database?

Reply:
I agree with Dasha, she mentioned one of the reasons I chose to create empty DeleteOutputData class and use it. The other reason is to stay more consistent in structure with other use cases.